content/en/platform/corda/5.1/developing-applications/notaries/_index.md

```diff
@@ -65,11 +65,17 @@ R3 produces a standard non-validating notary server CPB. This only contains CPKs
 
 ### Application CPB
 
-It is not possible to provide a standard application CPB, because the contents of this depend on the how you write a CorDapp. You must decide which CPKs you will bundle together to provide your application CPB. This is comprised of one or more CPKs which provide the application functionality. However, you must also bundle the appropriate CPKs for the notary protocols your CorDapp will support.
+It is not possible to provide a standard application CPB, because the contents of this depend on the how you write a CorDapp.
+You must decide which CPKs to bundle together to provide your application CPB.
+This is comprised of one or more CPKs which provide the application functionality.
+However, you must also bundle the appropriate CPKs for the notary protocols your CorDapp supports.
 
-The decision on which protocols to support is trivial at present, given there is only a single notary protocol to choose; the decision is purely based on whether the CorDapp is using the UTXO ledger model or not. If it is, then you must bundle the `notary-common`, `non-validating-notary-api` and `non-validating-notary-client` CPKs when creating your CPB. Otherwise, there is no requirement to bundle any additional CPKs (as the notary is not involved in flow only or consensual ledger based CorDapps).
+The decision on which protocols to support is trivial at present, given there is only a single notary protocol to choose; the decision is purely based on whether the CorDapp is using the UTXO ledger model or not.
+If it is, then you must bundle the `notary-common`, `non-validating-notary-api` and `non-validating-notary-client` CPKs when creating your CPB.
+Otherwise, there is no requirement to bundle any additional CPKs (as the notary is not involved in flow only or consensual ledger based CorDapps).
```

Comment:
```suggestion
Otherwise, there is no requirement to bundle any additional CPKs (as the notary is not involved in flow only or consensual ledger-based CorDapps).
```